core/core_bind.cpp

```diff
@@ -1428,29 +1288,13 @@ namespace special {
 PackedStringArray ClassDB::get_class_list() const {
 	List<StringName> classes;
 	::ClassDB::get_class_list(&classes);
-
-	PackedStringArray ret;
-	ret.resize(classes.size());
-	int idx = 0;
-	for (const StringName &E : classes) {
-		ret.set(idx++, E);
-	}
-
-	return ret;
+	return PackedStringArray(Vector<StringName>(classes));
```

Comment:
This could be slower than the previous implementation, because it needs two allocations.

Reply:
That's fair. Some conversion constructors which *also* change the wrapped type would solve that, I'll do that my next rebase